Python/optimizer_bytecodes.c

```diff
@@ -1096,6 +1096,18 @@ dummy_func(void) {
         sym_set_const(callable, len);
     }
 
+    op(_BINARY_SLICE, (container, start, stop -- res)) {
+        // Slicing a string always returns a string.
+        // TODO: We can apply this to lists and tuples as well.
+        //       We'll start with string to simplify the process.
```

Comment:
Wanna just add the others to this PR too? It's just two lines here and a couple of tests.